File: snippets/code/FishHave/bubble_sort.java
public static <Type>[] bubbleSort(<Type>[] array) {
    int len = array.length;
    
    for (var i = 0; i < len - 1; i++) {
        for (var j = 0; j < len - 1 - i; j++) {
            if (arr[j] > arr[j + 1]) {
                <Type> temp = arr[j+1];
                arr[j+1] = arr[j];
                arr[j] = temp;
            }
        }
    }

    return array;
}

File: snippets/code/FishHave/bubble_sort.go
func sort(arr int[]) int[] {
	len := len(arr);

    for i := 0; i < len - 1; i++ {
        for j := 0; j < len - 1 - i; j++ {
            if arr[j] > arr[j + 1] {
                var temp := arr[j+1];
                arr[j+1] = arr[j];
                arr[j] = temp;
        }
    }

    return arr;
} 